ebean-core/src/main/java/io/ebeaninternal/server/deploy/BeanDescriptorCacheHelp.java

```diff
@@ -294,8 +294,13 @@ boolean manyPropLoad(BeanPropertyAssocMany<?> many, BeanCollection<?> bc, Object
 
     List<Object> idList = entry.getIdList();
     bc.checkEmptyLazyLoad();
+    int i = 0;
     for (Object id : idList) {
-      many.add(bc, targetDescriptor.createReference(readOnly, id, persistenceContext));
+      final EntityBean ref = targetDescriptor.createReference(readOnly, id, persistenceContext);
+      if (many.hasOrderColumn()) {
+        ref._ebean_getIntercept().setSortOrder(++i);
```

Comment:
Ha, are we 1 based on sortOrder - interesting. Note to just double check this afterwards.

Reply:
Yeah we are! Confused me a bit when we started to use @OrderColumn, but was also more helpful in other places. Note: The official [documentation](https://docs.jboss.org/hibernate/jpa/2.2/api/javax/persistence/OrderColumn.html) states that the first element starts with 0 so this might not be what we want for people adopting Ebean and moving from another JPA implementation.